sparsity/dask/io.py

```diff
@@ -132,4 +146,36 @@ def _npz_read_divisions(paths, level=None):
                 file=paths[i], div1=divisions[i], div2=divisions[i+1]
             ))
 
-    return divisions
\ No newline at end of file
+    return divisions
+
+
+def write_npz_metadata(writes, divisions, paths, fn,
+                       block_size, storage_options):
```

Comment:
Looks like `writes` is unused.

Reply:
It is unused but necessary for how I use this function with dask. We might pass metadata of writing individual files via this parameter later...